src/common/connectors/notify/notify.js

```diff
@@ -0,0 +1,84 @@
+import Wreck from '@hapi/wreck'
```

Comment:
Do we have Wreck's proxy agent set up too?

Reply:
Not sure we need to do anything here according to this: https://github.com/DEFRA/apha-apps-perms-case-mgmt-api?tab=readme-ov-file#proxy